### PR TITLE
Norwegian.java

### PR DESCRIPTION
**Pull request by Diabloxx**
Transelation for Tillerinobot in Norwegian

### CODE REVIEW

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+	 *            characters.
+	 * @param name
+	 *            the irc nick which could not be resolved
+	 * @return
+	 */
+	public String unresolvableName(String exceptionMarker, String name) {
+		return "Your name is confusing me. Are you banned? If not, pls check out [https://github.com/Tillerino/Tillerinobot/wiki/How-to-fix-%22confusing-name%22-error this page] on how to resolve it!"
+				+ " if that does not work, pls [https://github.com/Tillerino/Tillerinobot/wiki/Contact contact Tillerino]. (reference "
+				+ exceptionMarker + ")";
+	}
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: Can you completely remove line 108-124?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+		 * "Nichts leichter als das!", which translates literally to
+		 * "Nothing easier than that!", which refers to German being my first
+		 * language.
+		 * 
+		 * Tillerino
+		 * 
+		 * P.S. you can put a link to your profile into the line like this:
+		 * [https://osu.ppy.sh/u/2070907 Tillerino]
+		 */
+		return new Message("Så du liker meg slik som jeg er? :)");
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: this message shouldnt be translated directly, read the comments above it. You don't have to do the suggested thing of crediting yourself but translating it directly doesnt make sense imo

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+	@Override
+	public String isSetId() {
+		return "Denne refererer til et set med maps ikke en enkelt map.";
+	}
+	
+	@Override
+	public String getPatience() {
+		return "Et øyeblikk.";
+	}
+}
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: cuz im a total nitpick, can you add an end of file newline here?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+	 */
+	public String unresolvableName(String exceptionMarker, String name) {
+		return "Your name is confusing me. Are you banned? If not, pls check out [https://github.com/Tillerino/Tillerinobot/wiki/How-to-fix-%22confusing-name%22-error this page] on how to resolve it!"
+				+ " if that does not work, pls [https://github.com/Tillerino/Tillerinobot/wiki/Contact contact Tillerino]. (reference "
+				+ exceptionMarker + ")";
+	}
+
+	@Override
+	public String excuseForError() {
+		return "Beklager, det var en fin sekvens med enere og nullere så jeg ble litt distrahert. Hva var det du spurte om igjen?")
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: change that `)` to a `;` plz :wink:

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java (hunk 3 above)
Comment (Diabloxx) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: Im not a 100% sure what you mean. Can you elaborate?

Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: this: https://omk.pics/1/4aezc
basically the last line of the file doesnt end in a newline, it really doesnt matter that much, its just bugging my ocd xd

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+/**
+ * TRANSLATION NOTE:
+ * 
+ * Please put some contact data into the following tag. If any additional
+ * messages are required, I'll use the English version in all translations and
+ * notify the authors.
+ * 
+ * @author Akuma Kudasai aka _BibleThump diabloxx50@gmail.com https://git.blosu.net/Diabloxx osu account is restricted
+ */
+public class Default implements Language {
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: welp, completely didnt see this, please change `Default` to `Norwegian`

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+	@Override
+	public String isSetId() {
+		return "Denne refererer til et set med maps ikke en enkelt map.";
+	}
+	
+	@Override
+	public String getPatience() {
+		return "Et øyeblikk.";
+	}
+} //end of file
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java]: nuuu, i didnt mean this :(
then again, I probably shouldnt be complaining about this, but what I meant, was just press the enter key once in your editor so it has a new empty line at the end of the file

**Pull request merged**